homework_1/task_02/taxes/taxes_test.go

```diff
@@ -0,0 +1,28 @@
+package taxes_test
+
+import (
+	"code-cadets-2021/homework_1/task_02/taxes"
+	"fmt"
+	"math"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestCalculateTax(t *testing.T) {
+	percentages := []float64{0.0, 0.10, 0.20, 0.30}
+	amounts := []float64{1000, 5000, 10000, math.Inf(1)}
```

Comment:
This should be part of the input in the test cases, shouldn't be constant across all test cases.